app/game/middleware/server-pipe.js

```diff
@@ -40,17 +40,17 @@ function getAction(message) {
         case MESSAGE_NAME.roundCountdownChanged:
             return RoundActionsCreator.updateRemaining(message.roundCountdown);
         default:
-            warn('Unknown message from server');
+            return null;
     }
-
-    return null;
 }
 
 function handleServerMessage(message, dispatch) {
```

Comment:
`handleServerMessage` uses `getAction` from closure, so shared `handleServerMessage`  function should get action as argument but it's equals to `dispatch + if` :)
For my personal opinion, shared method for this is overhead.